### PR TITLE
enable `multipleInstancesEnabled` in the management-ingress & ingress-nginx CR

### PR DESCRIPTION
**Pull request by ZhuoxiLi**
Tested in single-instance env & multi-instance env, both worked as expected.

### CODE REVIEW

**Review by Daniel-Fan — APPROVED** [review]
It looks good to me.

But I am wondering whether we could gather all the templates into one package like `constant`. Currently we already have several directories that only contain the constant template.

File: controllers/multipleInstancesEnabled/multipleInstancesEnabled.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+const MultipleInstancesEnabledTemplate = `
+- name: ibm-management-ingress-operator
+  spec:
+    managementIngress:
+      multipleInstancesEnabled: placeholder
+- name: ibm-ingress-nginx-operator
+  spec:
+    nginxIngress:
+      multipleInstancesEnabled: placeholder
```
Comment (horis233) [inline controllers/multipleInstancesEnabled/multipleInstancesEnabled.go:27]: We don't need to set it for `ibm-ingress-nginx-operator`

Comment (ZhuoxiLi) [reply controllers/multipleInstancesEnabled/multipleInstancesEnabled.go]: removed

**Review by horis233 — APPROVED** [review]
/lgtm

**Pull request merged**